plugins/engines/ospray/OSPRayScene.cpp

```diff
@@ -43,11 +43,12 @@ struct TextureTypeMaterialAttribute
     std::string attribute;
 };
 
-static TextureTypeMaterialAttribute textureTypeMaterialAttribute[7] = {
+static TextureTypeMaterialAttribute textureTypeMaterialAttribute[8] = {
     {TT_DIFFUSE, "map_kd"},
     {TT_NORMALS, "map_bump"},
+    {TT_BUMP, "map_bump"},
     {TT_SPECULAR, "map_ks"},
-    {TT_EMISSIVE, "map_a"},
+    {TT_EMISSIVE, "map_Ns"},
```

Comment:
Any reason why this one has capital N, when all other attributes are lowercase? convention? gltf?

Reply:
no, ospray. but they also accept lower-case 'n'. I'll change it